test/mocha.opts

```diff
@@ -0,0 +1 @@
+--file ./test/setup-sigint-listener.js
```

Comment:
I don't think you wanted to commit this.

Reply:
Well, since we removed the SIGINT listener, we need to have one in case that one presses CTRL-C during tests...

Reply:
Sorry, missed the `--file` part.